CommandHelper/LocalPackages/commands/buffs/howl.ms

```diff
@@ -0,0 +1,31 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': '§4H§co§7wl',
+            'class': 'howl',
+            'sig': 'oZusQq/JJEJYFtEk9o39UMTHlKygTYciYQc7cnPpogKcgexM84q8gkLnNvU9l7XR3Bd+KQK56FAYnkWP9C4V51jRTUyjLBKalMxbAQSUkGQjCphNoElK8QZG0/Vzt9TeSkVflMqlpkduKZQMgRqnsa2PSCoE/YJmoHNva2mr5JsKg35425ZILUe6DkENOxUqtCW6ni7QV1S0nIN8IXNJzy/05/G42T4Ml4ASYk4QDHBmOuf5w5NBir34qI0VDJC9d6LOHReWzTIfxwsmQc8/6cHRgWMJtkH3DK1Xnd51/LfIkjgxG8udWw0BzYnDSTrgFCpkQ8EoGR77IGzZznFWibTSQ3qNyCv8UJ2AZjZ3pyubsuFteEj4rqnsyR3dodFbXedOuavlMYj0CIUnG/eRkdR3YahvGJdZLxA7cbAeTYU2nUGs5jd9QjRpZH89PHW+5n497foWC+PPIfTSzEnkJu+qliSo3q9OZU56XNkK2efVCL5imncKRFJBuQcVJmaG6LvpDP+83xMoP51sPmrXFCg1IX3wdTAMZxk3+5BJ07nVYnVWYgR8KFVhIvIzKRZd8PGw+m8tQwFmmJQbY8OoAk0+/G5jHbZyE6hRrXHTDxPtIeZ10XGCmk/olSiA9qTStUCb80AxDGDRWfN75XphHtlkWRQfnAyyOAZhZf60owo=',
+            'val': 'eyJ0aW1lc3RhbXAiOjE0OTAyMzk0NDczMTIsInByb2ZpbGVJZCI6IjVkY2Y0NTcyNmNiYTQ3N2U5MjgxN2JkZGQ3ZWM1Njc2IiwicHJvZmlsZU5hbWUiOiJXeW5kZmFsbF8iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJtZXRhZGF0YSI6eyJtb2RlbCI6InNsaW0ifSwidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS84OGY3NjRjNDM3ODM3NDM2ODdkOTYzMjkxYWUwNDllNDU3NjA2NDBkZTM4ODMxMTc4Yzk5NWNiN2Q1YTViNGIifX19',
+            'trail': '',
+            'op': 'false',
+            'speech': "All this is just sorcery to keep the Witch of the West away.",
+            'powers': array(
+                "/buff",
+                "/jump 0-5",
+                "/run 0-3",
+                "/haste 0-5",
+                "/fly",
+                "/flyspeed <number>",
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/disguise"
```

Comment:
Proof for carry player, jump, run and fly:http://video.disney.com/watch/that-s-my-girl-4d91a69e7a08fad88e31f9ed
Proof for flyspeed:https://www.youtube.com/watch?v=UibodUGoL4M at 1:24
Proof for disguise: http://www.wallpapersxl.com/wallpapers/1680x945/howl/181248/howl-181248.jpg (This is the disguise howl uses to watch over Sophie while she visits solomon for Howl to tell her that he will not come.